packages/containers/src/ActionBar/ActionBar.connect.js

```diff
@@ -0,0 +1,36 @@
+import { cmfConnect } from '@talend/react-cmf';
+import { ActionBar } from '@talend/react-components';
+import getRenderers from '../renderers';
+
+export function mapStateToProps(state, { actionIds }) {
+	const props = {
+		renderers: getRenderers(),
+	};
+	if (actionIds) {
+		props.actions = {};
+		const { left, right } = actionIds;
+		if (left) {
+			props.actions.left = left.map((actionId) => {
+				if (typeof actionId === 'string') {
+					return { actionId };
+				}
+				return actionId;
+			});
+		}
+		if (right) {
+			props.actions.right = right.map(actionId => ({ actionId }));
```

Comment:
shouldn't this be the same as left on line https://github.com/Talend/ui/pull/822/files#diff-4437e321186a4aadc6fe31a004d48f98R13